ansible/roles_ocp_workloads/ocp4_workload_shared_cluster_access/tasks/post_workload.yml

```diff
@@ -1,9 +1,32 @@
 ---
 # Implement your Post Workload deployment tasks here
-# --------------------------------------------------
+#--- --------------------------------------------------
+- name: Get console route
+  command: oc get route -n openshift-console console -o json
+  register: routeconsole
 
+- name: API for CLI
+  command: oc whoami --show-server
```

Comment:
same for this

Reply:
AFAIK, there is no API endpoint to get the server address so can't be obtained through k8s_info. If you know where can be found, let's talk.

Reply:
Hmm, I think you're right. My mistake.